aiocassandra.py

```diff
@@ -13,7 +13,7 @@
 except ImportError:
     import trollius as asyncio
 
-__version__ = '1.1.0'
+__version__ = '1.1.1dev2'
```

Comment:
@hellysmile `setup.get_version()` is unable to correctly parse this :(

Reply:
Nice catch!